ospfd/ospf_vty.c

```diff
@@ -10755,10 +10755,11 @@ DEFUN (ospf_route_aggregation_timer,
 
 DEFPY (show_ip_ospf_gr_helper,
        show_ip_ospf_gr_helper_cmd,
-       "show ip ospf [vrf <NAME|all>] graceful-restart helper [detail] [json]",
+       "show ip ospf [(1-65535)]$instance [vrf <NAME|all>] graceful-restart helper [detail] [json]",
```

Comment:
I'm pretty sure that instance cannot be mixed with vrf.  They are orthogonal.  So shouldn't that imply then that it could be instance or vrf?

Reply:
@donaldsharp @odd22

Reply:
But , this command can be executed successfully:
```show ip ospf 1 vrf default  database  detail  json```